examples/gpt3/config.yaml

```
@@ -0,0 +1,21 @@
+data_dir: data
+environment_variables: {}
+examples_filename: examples.yaml
+input_type: Any
+model_class_filename: model.py
+model_class_name: GPT3Model
+model_framework: custom
+model_metadata: {}
+model_module_dir: model
+model_name: null
+model_type: custom
+python_version: py39
+requirements:
+  - openai==0.6.3
+resources:
+  cpu: 500m
+  memory: 512m
+  use_gpu: false
+secrets:
+  api_key:
```

since this will be read from a bunch of secrets configured in baseten, `api_key` won't be clear what it is without the context of the model it's for
```suggestion
  openai_api_key:
```